stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api/ChatApi.kt

```diff
@@ -65,100 +66,49 @@ internal class ChatApi(
     fun setConnection(userId: String, connectionId: String) {
         this.userId = userId
         this.connectionId = connectionId
+        fileUploader.setConnection(userId, connectionId)
```

Comment:
`setConnection` is a bit weird on the `FileUploader` interface.
Maybe we could move those arguments to `send()`, `delete()` methods

Reply:
I was thinking about it. We need it in `userId` and `connectionId` in our implementation but I'm not sure if user's implementations are going to need that. The question is if it's better to add an additional method to the interface or add those parameters to every method even if they won't be used by other users?

Reply:
I would prefer to add all argument to the methods, then they don't need to know the internal lifecycle that we use with this class.
If they don't need this argument, they can ignore it